src/plugins/intel_gpu/src/kernel_selector/kernels/scatter_update/scatter_nd_update_kernel_ref.cpp

```diff
@@ -184,12 +184,27 @@ KernelsData ScatterNDUpdateKernelRef::GetKernelsData(const Params& params, const
             size_t input2_rank = newParams.inputs[2].LogicalDims().size();
             cldnn_jit.AddConstant(MakeJitConstant("IS_SECOND_ITER", "true"));
             size_t shape_info_offset = 0;
-            cldnn_jit.AddConstant(MakeJitConstant("INPUT0_BLOCK_ND", GetInputBlockND(newParams, 0, shape_info_offset, input0_rank)));
-            if (newParams.inputs[0].is_dynamic())
-                shape_info_offset++;
-            cldnn_jit.AddConstant(MakeJitConstant("INPUT1_BLOCK_ND", GetInputBlockND(newParams, 1, shape_info_offset, newParams.indices_rank - 1)));
-            if (newParams.inputs[1].is_dynamic())
-                shape_info_offset++;
+            cldnn_jit.AddConstant(
+                MakeJitConstant("INPUT0_BLOCK_ND", GetInputBlockND(newParams, 0, shape_info_offset, input0_rank)));
+            if (newParams.inputs[0].is_dynamic()) {
+                shape_info_offset += DataTensor::max_rank();
```

Comment:
Does it differ from GetDynamicShapeOffset() value?

Reply:
Thats same. I fixed, thanks.